rest/CMakeLists.txt

```diff
@@ -28,17 +28,19 @@ etn_target(shared ${PROJECT_NAME}-rest
         ${PROJECT_NAME}-common
         fty-asset-libng
         fty_common_messagebus
-    TARGET_DESTINATION /usr/lib/bios
+    TARGET_DESTINATION
```

Comment:
Are you sure ? by default CMAKE_INSTALL_LIBDIR=lib

Reply:
yes LIBDIR debian == /usr/lib/x86....